NeuroID/src/advancedDeviceAndroidLib/java/com/neuroid/tracker/extensions/AdvancedDeviceExtension.kt

```diff
@@ -25,13 +26,18 @@ fun NeuroID.start(advancedDeviceSignals: Boolean) {
             val keyService = NIDAdvKeyService()
             var fpjsRetryCount = 0
             val FPJS_RETRY_MAX = 3
+            var endpointUrl = Constants.fpjsDevDomain.displayName
+            if (NeuroID.getInstance()?.getEnvironment() == "LIVE"){
+                endpointUrl = Constants.fpjsProdDomain.displayName
+            }
```

Comment:
This is my fault for putting it in the ticket (I was making it quick). Although there is a dev option, we should not be putting that in the SDK as our customers regardless of their `env` will only ever call PROD. Only we as internal NID devs will call the dev endpoint.

Reply:
Ok. I removed the dev endpoint completely from the sdk.

Reply:
Should we document the dev endpoint in the readme?

Reply:
Adding the comment in the Jira ticket for documentation.

Reply:
Tested with site id and key for adv enabled form_1oLmFMjl and its working as expected.